schemas/pagenation.py

```
@@ -0,0 +1,15 @@
+from typing import TypeVar, Generic, List
```

Ruff (0.14.11)

`typing.List` is deprecated, use `list` instead

(UP035)